service/frontend/workflow_handler.go

```diff
@@ -3066,6 +3066,7 @@ func (wh *WorkflowHandler) CreateSchedule(ctx context.Context, request *workflow
 		WorkflowIdConflictPolicy: enumspb.WORKFLOW_ID_CONFLICT_POLICY_FAIL,
 		Memo:                     request.Memo,
 		SearchAttributes:         sa,
+		Priority:                 &commonpb.Priority{}, // ie default priority
```

Comment:
Added a default priority where there's no custom one to make it easy to find them all.

Reply:
I guess I'd rather leave it nil? but either way is fine here

Reply:
I like specifying it because it avoids the ambiguity of "is priority been considered here - or is it missing?"